### PR TITLE
feat: add code for alizer npm package

### LINKED ISSUE #20: Add Javascript implementation

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #20 

cc @evidolob

### CODE REVIEW

File: js/src/recognizer.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        glob(`${root}/**/*.*`, `!${root}/**`, (err: any, files: string[] | PromiseLike<string[]>) => {
+            if (err) {
+                reject(`Unable to retrieve files from current project ${root}. ${err}`);
+            }
+            return resolve(files);
+        })
+    });
+}
+
+detectLanguages('/home/luca/Public/github.com/redhat-developer/empty-dir');
```
Comment (jeffmaury) [inline js/src/recognizer.ts:90]: Hard code path !!!!

**Review by jeffmaury — CHANGES_REQUESTED** [review]
See comments and

- nice to have a few unit tests
- language.yml file duplicated, at least open an issue to share it between implementations
- add or edit workflows

**Comment by lstocchi** [comment]
> See comments and
> 
> * nice to have a few unit tests
> * language.yml file duplicated, at least open an issue to share it between implementations
> * add or edit workflows

I just copied everything as it is here so yevhen can start working with it. I'll open new issues for the second/third point

**Review by evidolob — APPROVED** [review]

**Comment by jeffmaury** [comment]
> 
> 
> > See comments and
> > 
> > * nice to have a few unit tests
> > * language.yml file duplicated, at least open an issue to share it between implementations
> > * add or edit workflows
> 
> I just copied everything as it is here so yevhen can start working with it. I'll open new issues for the second/third point

Point 3 is required for us to validate changes

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
+  build-java:
```
Comment (jeffmaury) [inline .github/workflows/CI.yml:12]: Seems the name of the workflow needs to be renamed as well

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by lstocchi** [comment]
All yours @jeffmaury

**Pull request merged**